components/ai_chat/resources/page/components/svg/caret.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from "react"
+
+const SvgComponent = (props: any) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={3}
+    height={12}
+    fill="none"
+    {...props}
+  >
+    <rect width={3} height={12} fill="#000" rx={1} />
+  </svg>
```

Comment:
We could make this blink too. If we did, it would be wise to disable the animation with `@media (prefers-reduced-motion)`.